src/Compilers/Core/Portable/Emit/Context.cs

```diff
@@ -19,5 +20,27 @@ public EmitContext(CommonPEModuleBuilder module, SyntaxNode syntaxNodeOpt, Diagn
             SyntaxNodeOpt = syntaxNodeOpt;
             Diagnostics = diagnostics;
         }
+
+        public bool Filter(IMethodDefinition method)
```

Comment:
The trouble is that this is needed both in FullMetadataWriter and MetadataVisitor, which don't share a base type.
I could implement extension methods on `IMethodDefinition` and `ITypeDefinitionMember`. Do you think that's better?